frontends/open-discussions/src/components/PageBanner.js

```diff
@@ -0,0 +1,178 @@
+import React from "react"
```

Comment:
No changes here besides removing TS definitions. This is generally the same as `ol-util/src/components/PageBanner.ts`.